configs/ote/custom-object-detection/mobilenetV2_ATSS/template.yaml

```diff
@@ -21,6 +21,7 @@ application:
 entrypoints:
   base: mmdet.apis.ote.apis.detection.OTEDetectionTask
   openvino: mmdet.apis.ote.apis.detection.OpenVINODetectionTask
+  nncf: mmdet.apis.ote.apis.detection.NNCFDetectionTask
```

Comment:
Did we planned to have NNCF* tasks? This template file cannot be read by the SDK.